pubsub/pubsub.go

```diff
@@ -175,8 +182,10 @@ func newTopic(d driver.Topic) *Topic {
 			dms = append(dms, dm)
 		}
 
-		return retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() error {
-			return d.SendBatch(callCtx, dms)
+		return retry.Call(callCtx, gax.Backoff{}, d.IsRetryable, func() (err error) {
+			ctx := trace.StartSpan(callCtx, "gocloud.dev/pubsub.SendBatch")
```

Comment:
pubsub/driver.Topic.SendBatch?

Reply:
This is a driver call, not a concrete type call, do we even need to trace it? The end user doesn't know that this API exists...

Or at least distinguish it from the concrete type traces somehow, e.g. `pubsub.driver.SendBatch`.